Dockerfile

```diff
@@ -1,66 +1,97 @@
+FROM ubuntu:18.04 AS builder
+
+LABEL maintainer="team@appwrite.io"
+
+ARG TESTING=false
+
+ENV TZ=Asia/Tel_Aviv \
+    DEBIAN_FRONTEND=noninteractive \
+    PHP_VERSION=7.3 \
+    PHP_REDIS_VERSION=3.1.2
+
+RUN \
+  apt-get update && \
+  apt-get install -y --no-install-recommends --no-install-suggests ca-certificates software-properties-common wget curl git openssl && \
+  LC_ALL=C.UTF-8 add-apt-repository -y ppa:ondrej/php && \
+  apt-get update && \
+  apt-get install -y --no-install-recommends --no-install-suggests make php$PHP_VERSION php$PHP_VERSION-dev zip unzip php$PHP_VERSION-zip && \
```

Comment:
is the zip & unzip required because of --no-install-recommends?

Reply:
Yes because in this case, we install what we need or what is actually required as a dependency.